### PR TITLE
Adds initial setup code

### PR DESCRIPTION
**Pull request by jeffreypaul15**
Adds the initial setup code for plotting.
I've moved the creation of the various meshes (quadrangle and map) to `draw.py`, I thought that separating the code for creating and adding the meshes would be better.
`pv.start_xvfb()` is required for the segmentation faults to not occur but this doesn't really fix the tests from not executing though, I will have to look into this.

### CODE REVIEW

File: sunkit_pyvista/version.py [hunk 1]
```diff
@@ -5,4 +5,4 @@
     from setuptools_scm import get_version
     __version__ = get_version(root='..', relative_to=__file__)
 except Exception:
-    __version__ = '0.1.dev4+g7f5b6c0.d20210519'
+    __version__ = '0.1.dev17+g3b166a0.d20210526'
```
Comment (nabobalis) [inline sunkit_pyvista/version.py:8]: This should be "0.0.0" or something from what I recall.

File: tox.ini [hunk 2]
```diff
@@ -28,9 +28,10 @@ description =
 
 deps =
 # The following indicates which extras_require from setup.cfg will be installed
+    scipy
```
Comment (nabobalis) [inline tox.ini:31]: Why is this here when its part of the setup.cfg?

Comment (jeffreypaul15) [reply tox.ini:31]: Sorry, this was a local virtualenv issue

File: sunkit_pyvista/version.py (hunk 1 above)
Comment (dstansby) [reply sunkit_pyvista/version.py]: Should this file even be here, or should it be gitignored? I can fix this in another PR either way.

File: setup.cfg [hunk 3]
```diff
@@ -16,7 +16,10 @@ setup_requires = setuptools_scm
 install_requires =
     sunpy>=3
     pyvista
-
+    matplotlib
```
Comment (dstansby) [inline setup.cfg:19]: I don't think Matplotlib should need to be an installation requirement, do you need it for anything at the moment?

File: setup.cfg [hunk 4]
```diff
@@ -16,7 +16,10 @@ setup_requires = setuptools_scm
...
+    scipy
+    pyvirtualdisplay
```
Comment (dstansby) [inline setup.cfg:21]: Similarly I don't think this should be an install requirement?

File: sunkit_pyvista/version.py (hunk 1 above)
Comment (nabobalis) [reply sunkit_pyvista/version.py]: Hmm, I think this file should gitignore.

File: sunkit_pyvista/plotter.py [hunk 5]
```diff
@@ -1,10 +1,138 @@
...
+        return self._plotter
+
+    @functools.wraps(pv.Plotter.show)
+    def show(self, *args, **kwargs):
+        """
+        Show the plot.
+        """
+        self.plotter.show(*args, **kwargs)
+
+    def _pyvista_mesh(self, m):
```
Comment (dstansby) [inline sunkit_pyvista/plotter.py:49]: I think this should live in a separate file as a function (not a method) that takes the signature
```python
def pyvista_mesh(smap, coordinate_frame):
```

File: sunkit_pyvista/draw.py [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+from sunpy.map.maputils import all_corner_coords_from_map
+
+from matplotlib.patches import Polygon
+from astropy.constants import R_sun
+
+
+__all__ = ['Draw']
+
+
+class Draw:
```
Comment (dstansby) [inline sunkit_pyvista/draw.py:19]: I don't think I understand why this is a class? I think it would be best to get remove this for now, and just focus on getting the `plot_map` method working. We can add other stuff later once that bit is working.

File: sunkit_pyvista/draw.py [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+        return self._coordinate_frame
+
+    def _convert_to_Skycoord(self, m, coords, frame='heliographic_stonyhurst'):
+        """
+        Converts a given set of coordinates to SkyCood objects with
+        """
+        corners = SkyCoord(coords.xy[:,0]*u.deg, coords.xy[:,1]*u.deg , frame=frame, obstime=m.date)
+        return corners
+
+    def draw_quadrangle(self, m, bottom_left, *, width: u.deg = None, height: u.deg = None,
```
Comment (dstansby) [inline sunkit_pyvista/draw.py:39]: I think there is a better way to implement this method, but again lets remove it from this PR for now and focus on `add_map`.

File: sunkit_pyvista/plotter.py [hunk 8]
```diff
@@ -1,10 +1,138 @@
...
+        """
+        `pyvista.Plotter`.
+        """
+        return self._plotter
+
+    @functools.wraps(pv.Plotter.show)
+    def show(self, *args, **kwargs):
+        """
+        Show the plot.
+        """
```
Comment (dstansby) [inline sunkit_pyvista/plotter.py:46]: I think there shouldn't be a docstring here, so the method can automatically inherit the docstring from `pyvista.Plotter.show`.

File: tox.ini [hunk 9]
```diff
@@ -30,7 +30,7 @@ deps =
 # The following indicates which extras_require from setup.cfg will be installed
 extras =
     test
-
+    all
```
Comment (dstansby) [inline tox.ini:33]: Why have you added this?

Comment (jeffreypaul15) [reply tox.ini:33]: I was facing a local venv issue, sorry I'll fix this

File: sunkit_pyvista/draw.py (hunk 6 above)
Comment (jeffreypaul15) [reply sunkit_pyvista/draw.py:19]: Ah, okay. 
I thought of splitting the code for generating pyvista meshes and the addition of them to the plotter separately

File: sunkit_pyvista/plotter.py (hunk 5 above)
Comment (jeffreypaul15) [reply sunkit_pyvista/plotter.py:49]: Okay, sure. I'll restructure this PR with the initial code that you wrote and make another one with the splitting of the code into separate classes and files.

File: setup.cfg (hunk 3 above)
Comment (jeffreypaul15) [reply setup.cfg:19]: Not at the moment, I'll remove the draw_quadrangle method so I'll get rid of this as well.

File: setup.cfg (hunk 4 above)
Comment (jeffreypaul15) [reply setup.cfg:21]: yes, this was a venv issue sorry again
